cityofaustin/atd-vz-data
# removes micromobility device flag

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/12012

this is the last step in dropping the `micromobility_device_flag` column from vzd and representing it as a mode like all others. this was so straightforward thanks to @mddilley's amazing work setting up hasura migrations on vzd! this should also be really simple to test and integrate into the next release. the automated migrations are already paying off!

## Testing
**URL to test:** 
local

**Steps to test:**
- start up a local instance of vze
- go to any crash page - the 'Micromobility device flag' column should no longer appear in the flags table
- connect to your local database using a sql client or `./vision-zero hasura-console`
- open the `atd_txdot_crashes` table and note that the `micromobility_device_flag` column no longer appears

---
#### Ship list
